t/Net/DRI/Protocol/EPP/Extensions/UniRegistry.t

```diff
@@ -178,12 +178,12 @@ is_deeply($rc->get_data('session','server','extensions_selected'),['urn:ietf:par
 is($rc->is_success(),1,'ICM - session login is_success');
 # enforce local extensions
 $R2='';
 $rc=$dri->process('session','login',['ClientX','foo-BAR2',{client_newpassword => 'bar-FOO2', only_local_extensions => 1 }]);
-is($R1,$E1.'<command><login><clID>ClientX</clID><pw>foo-BAR2</pw><newPW>bar-FOO2</newPW><options><version>1.0</version><lang>en</lang></options><svcs><objURI>urn:ietf:params:xml:ns:domain-1.0</objURI><objURI>urn:ietf:params:xml:ns:host-1.0</objURI><objURI>urn:ietf:params:xml:ns:contact-1.0</objURI><objURI>http://ns.uniregistry.net/eps-1.0</objURI><svcExtension><extURI>http://ns.uniregistry.net/centric-1.0</extURI><extURI>http://ns.uniregistry.net/market-1.0</extURI><extURI>urn:afilias:params:xml:ns:association-1.0</extURI><extURI>urn:ietf:params:xml:ns:fee-0.7</extURI><extURI>urn:ietf:params:xml:ns:idn-1.0</extURI><extURI>urn:ietf:params:xml:ns:launch-1.0</extURI><extURI>urn:ietf:params:xml:ns:rgp-1.0</extURI><extURI>urn:ietf:params:xml:ns:secDNS-1.1</extURI></svcExtension></svcs></login><clTRID>ABC-12345</clTRID></command>'.$E2,'ICM - session login build only_local_extensions = 1');
+is_string($R1,$E1.'<command><login><clID>ClientX</clID><pw>foo-BAR2</pw><newPW>bar-FOO2</newPW><options><version>1.0</version><lang>en</lang></options><svcs><objURI>urn:ietf:params:xml:ns:domain-1.0</objURI><objURI>urn:ietf:params:xml:ns:host-1.0</objURI><objURI>urn:ietf:params:xml:ns:contact-1.0</objURI><objURI>http://ns.uniregistry.net/eps-1.0</objURI><svcExtension><extURI>http://ns.uniregistry.net/centric-1.0</extURI><extURI>http://ns.uniregistry.net/market-1.0</extURI><extURI>http://www.verisign.com/epp/sync-1.0</extURI><extURI>urn:afilias:params:xml:ns:association-1.0</extURI><extURI>urn:ietf:params:xml:ns:fee-0.7</extURI><extURI>urn:ietf:params:xml:ns:idn-1.0</extURI><extURI>urn:ietf:params:xml:ns:launch-1.0</extURI><extURI>urn:ietf:params:xml:ns:rgp-1.0</extURI><extURI>urn:ietf:params:xml:ns:secDNS-1.1</extURI></svcExtension></svcs></login><clTRID>ABC-12345</clTRID></command>'.$E2,'ICM - session login build only_local_extensions = 1');
```

Comment:
see note elsewhere, a change to VeriSign::Sync module makes it automatically load even when local_only_extensions. Not a biggue

Reply:
Should be fine for now but when all PM versions are merged we should double check and test properly those `only_local_extensions`: `Core.t` and `UniRegistry.t`